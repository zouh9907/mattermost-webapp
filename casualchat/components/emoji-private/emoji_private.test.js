// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.
import React from 'react';
import {shallow} from 'enzyme';

import EmojiPrivate from './emoji_private';

describe('casualchat/components/emoji-private', () => {
    const baseProps = {
        teamId: 'team-id',
        teamName: 'team-name',
        teamDisplayName: 'team-display-name',
        siteName: 'site-Name',
        scrollToTop: jest.fn(),
        actions: {
            loadRolesIfNeeded: jest.fn(),
        },
    };
    test('should match snapshot', () => {
        const wrapper = shallow(<EmojiPrivate {...baseProps}/>);
        expect(wrapper).toMatchSnapshot();
    });
});
