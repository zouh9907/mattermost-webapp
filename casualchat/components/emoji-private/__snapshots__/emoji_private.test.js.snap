// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`casualchat/components/emoji-private should match snapshot 1`] = `
<div
  className="backstage-content emoji-list"
>
  <div
    className="backstage-header"
  >
    <h1>
      <FormattedMessage
        defaultMessage="Private Emoji"
        id="emoji_list.header-private"
        values={Object {}}
      />
    </h1>
    <Link
      className="add-link"
      to="/team-name/emoji_private/add_private"
    >
      <button
        className="btn btn-primary"
        type="button"
      >
        <FormattedMessage
          defaultMessage="Add Private Emoji"
          id="emoji_list.add-private"
          values={Object {}}
        />
      </button>
    </Link>
  </div>
  <Connect(EmojiList)
    isPrivate={true}
    scrollToTop={[MockFunction]}
  />
</div>
`;
